Copy edits and MS comments: Part 2 (#267)
AlexsLemonade/OpenPBTA-manuscript: content/03.results.md

#### Prediction of _TP53_ oncogenicity and telomerase activity

To understand the _TP53_ phenotype in each tumor, we ran a classifier previously trained on TCGA [@doi:10.1016/j.celrep.2018.03.076] to infer _TP53_ inactivation status.
We compared results of this classifier to "true positive" alterations derived using high-confidence SNVs, CNVs, SVs, and fusions in _TP53_.

> Specifically, we annotated _TP53_ alterations as "activated" if samples harbored one of p.R273C or p.R248W mutations [@doi:10.1038/ng0593-42], or "lost" if the originating patient either had a Li Fraumeni Syndrome (LFS) predisposition diagnosis, the sample harbored a known hotspot mutation, or the sample contained two hits (e.g. both SNV and CNV), which would suggesting both alleles had been affected.

Author: We can say "the given patient" instead of "originating"?

> Specifically, we annotated _TP53_ alterations as "activated" if samples harbored one of p.R273C or p.R248W mutations [@doi:10.1038/ng0593-42], or "lost" if the originating patient either had a Li Fraumeni Syndrome (LFS) predisposition diagnosis, the sample harbored a known hotspot mutation, or the sample contained two hits (e.g. both SNV and CNV), which would suggesting both alleles had been affected.

Reviewer: Sure, though I don't think the patient needs an adjective here, per se.

If the _TP53_ mutation did not reside within the DNA-binding domain or we did not detect any alteration in _TP53_, we annotate the tumor as "other," reflecting its unknown _TP53_ alteration status.

The classifier achieved a high accuracy (AUROC = 0.85) for rRNA-depleted, stranded samples compared to randomly shuffled _TP53_ scores (**Figure {@fig:Fig4}A**). 
By contrast, the classifier showed somewhat lower accuracy on poly-A samples (AUROC = 0.62; **Figure {@fig:S5}A**).
As this classifier has previously shown strong performance on poly-A data [@doi:10.1016/j.celrep.2019.09.071], the relatively smaller number of poly-A samples (N = 58) compared to stranded samples (N = 977) may explain the difference in performance across these two libraries.

> We further compared the _TP53_ scores calculated by the classifier.

Author: I actually added this because it was not previously clear to me that the scores were _from the classifier_.

> We further compared the _TP53_ scores calculated by the classifier.

Reviewer: >To understand the _TP53_ phenotype in each tumor, we ran a classifier previously trained on TCGA [@doi:10.1016/j.celrep.2018.03.076] to calculate a _TP53_ score and infer _TP53_ inactivation status.

should we change the wording here somehow?

Indeed, we observed higher scores in LFS tumors (N = 8) for which we detected high-confidence _TP53_ somatic alterations (**Tables S1 and S3**). 

Although we were unable to detect canonical somatic _TP53_ mutations in two LFS patient tumors with low _TP53_ scores (BS_DEHJF4C7 with a score of 0.09 and BS_ZD5HN296 with a score of 0.28), we confirmed the LFS diagnoses from pathology reports and found each to have a pathogenic germline variant in _TP53_.
In addition, the tumor purity of these two LFS samples was low (16% and 37%, respectively) **TODO: Where are these numbers from? I don't see in the metadata file.**, suggesting the classifier may require a certain level of tumor purity to achieve good performance, as we expect _TP53_ to be intact in normal cells. 
Tumors with the highest median _TP53_ scores were those known to harbor somatic _TP53_ alterations and included DMGs, medulloblastomas, HGGs, DNETs, ependymomas, and craniopharyngiomas (**Figure {@fig:Fig4}D**), while gangliogliomas, LGGs, meningiomas, and schwannomas had the lowest median scores.

> We next used gene expression data to predict telomerase activity using **EX**pression-based **T**elomerase **EN**zymatic activity **D**etection (EXTEND) [@doi:10.1038/s41467-020-20474-9] as a surrogate measure of malignant potential [@doi:10.1038/s41467-020-20474-9; 10.1093/carcin/bgp268], such that higher EXTEND scores suggest increase malignant potential.

Author: Ah, interesting point that this could allow for typos elsewhere. I can remove the bolding and associated dictionary items.

As expected, EXTEND scores significantly correlated with _TERC_ (R = 0.619, p < 0.01) and _TERT_ (R = 0.491, p < 0.01) expression (**Figure {@fig:S5}B-C**).  

> We found aggressive tumors such as **TODO: Not all of these are explicitly laid out in 4D - rephrase based on 4D?** CNS lymphoma, ETMR, ATRT, DMG, and HGG had high EXTEND scores (**Figure {@fig:Fig4}D**), while benign lesions such as teratomas, dysplasias, and hemangioblastomas had the lowest scores (**Table S3**).

Author: Can you help with this, just which ones I should list here? I'm just not sure which are technically "aggressive" here.. thanks!

> We found aggressive tumors such as **TODO: Not all of these are explicitly laid out in 4D - rephrase based on 4D?** CNS lymphoma, ETMR, ATRT, DMG, and HGG had high EXTEND scores (**Figure {@fig:Fig4}D**), while benign lesions such as teratomas, dysplasias, and hemangioblastomas had the lowest scores (**Table S3**).

Reviewer: Sure. Aggressive = DMG, HGG, certain MB subtypes (non-WNT, but not labeled). We can perhaps call attention to the fact that although MB contains aggressive and non-aggressive subgroups, they have high median scores with a tight range, suggesting all subgroups may have some dysfunctional p53. I actually looked at these by subgroup early on and didn't see notable differences, which was surprising, as there is an official SHH TP53-mutant subgroup now, but not for others! 

For non-aggressive, schwannoma, meningioma, LGG, GNG, DNET.

From patient PT_JNEV57VK, only one of the two cell lines derived from the progressive tumor was hypermutated (TMB = 35.9 Mut/Mb).
This hypermutated cell line had a strong exposure to signature 11, while this patient's non-hypermutated cell line showed a number of lesser exposures (1, 11, 18, 19, MMR2), highlighting the plasticity of mutational processes and the need to carefully genomically characterize and select models for preclinical studies based on research objectives.
We observed that signature 18, which has been associated with high genomic instability and can lead to a hypermutator phenotype [@doi:10.1038/s43018-020-0027-5], was uniformly represented among hypermutant solid tumors. 
Additionally, we found that all of these tumor or cell lines had dysfunctional _TP53_ (**Table 2**), consistent with a previous report showing _TP53_ dysregulation is a dependency of tumors with high genomic instability [@doi:10.1038/s43018-020-0027-5].

> With one exception, hypermutant and ultra-hypermutant tumors had high  _TP53_ scores (> 0.5) and telomerase activity **Figure {@fig:Fig4}D**. **TODO: 1/3 of the telomerase activities are <0.5, so what counts as "high" here?**

Author: > The lowest they used was 0.5

I'm not sure what this means, can you clarify? Who is they?

We might want to stick to just tp53 here without any cutoff for telomerase.

> With one exception, hypermutant and ultra-hypermutant tumors had high  _TP53_ scores (> 0.5) and telomerase activity **Figure {@fig:Fig4}D**. **TODO: 1/3 of the telomerase activities are <0.5, so what counts as "high" here?**

Reviewer: This was in reference to the authors of the EXTEND manuscript - they didn't state exactly what a high or low score was either, but used distributions for cutoffs, so we can just remove. That's fine. I was just trying to draw attention to the colored dots in the telomerase panel of the boxplot as well since we otherwise don't discuss, but maybe we can just skip here since the colored dots are noted in the legend.

This model identified several expected trends, including a significant overall survival benefit if the tumor had been fully resected (HR = 0.36, 95% CI = 0.2 - 0.67, p = 0.001) or if the tumor belonged to the LGG group (HR = 0.068, 95% CI = 0.0091 - 0.5, p = 0.008) as well as a significant risk if the tumor belonged to the HGG group (HR = 5.7, 95% CI = 3.5 - 9.0, p < 0.001).
High telomerase scores were a poor prognostic indicator across brain tumor histologies (HR = 24, 95% CI = 7.1 - 79, p < 0.001), demonstrating that EXTEND scores calculated from RNA-Seq are an effective rapid surrogate measure for telomerase activity.
Although higher _TP53_ scores, which would indicate _TP53_ gene or pathway dysregulation, were not a significant predictor of risk across the entire OpenPBTA cohort (**Table S4**), separate cox regressions constructed for individual cancer groups revealed significant survival risk for both DMGs (HR = 1.77e6, 95% CI = 1.98-1.57e12, p = 0.04) and ependymomas (HR = 1612, 95 % CI = 9.1-2.9e5, p = 0.005).
Since we observed the negative prognostic effect of _TP53_ scores for HGGs, we assessed the effect of molecular subtypes within HGG samples on survival risk.

> We found that DMG H3 K28 tumors with _TP53_ loss had significantly worse prognosis (HR = 3.2, CI = 1.5-6.6, p = 0.002) than did DMG H3 K28 tumors with wildtype _TP53_ (**Figure {@fig:Fig4}G** and **Figure {@fig:Fig4}H**).

Author: Grammatically, this needs the "did."

We observed that, except for three outliers, _C11orf95::RELA_ (_ZFTA::RELA_) fusion-positive ependymomas fell within distinct clusters (**Figure {@fig:S6}A**).
Medulloblastoma (MB) samples cluster by molecular subtype, with WNT and SHH in distinct clusters and Groups 3 and 4 showing some overlap (**Figure {@fig:S6}B**), as expected.
Of note, two MB samples annotated as the SHH subtype did not cluster with the other MB samples, and one clustered with Group 3 and 4 samples, suggesting potential subtype misclassification or different underlying biology of these two tumors.
_BRAF_-driven low-grade gliomas (**Figure {@fig:S6}C**) were present in three separate clusters, suggesting that there might be additional shared biology within each cluster.
Histone H3 G35-mutant high-grade gliomas generally clustered together and away from K28-mutant tumors (**Figure {@fig:S6}D**).
Interestingly, although H3 K28-mutant tumors have different biological drivers than do H3 wildtype tumors **TODO: Is there a reference for different biological drivers?**, they did not form distinct clusters. 

> This pattern suggests these subtypes may be driven by common transcriptional programs, have other much stronger biological drivers such as their known distinct epigenetic drivers **TODO: this phrase confuses me because I don't see how _distinct_ epigenetic drivers would cause clustering together?** , or our sample size is too small to detect transcriptional differences.

Reviewer: Ah, this is because of the origin of the H3 K28 mutation - it turns H3K27Me3 (repressive) to H3K27Ac (enhancer), thus more open chromatin, more accessibility for transcription to occur. I would have expected H3 wildtype and K28s to cluster apart based on this mutation, which causes an epigenetic modification, since the chromatin landscape of the tumors is _so_ drastically different. Are you saying that you don't understand why they would cluster together given this above diff? That is why it is surprising to me and this is just a list of possibilities, one of which is small sample size to detect a real effect.

Also, I think I should have said this:
```suggestion
This pattern suggests these subtypes may be driven by common transcriptional programs, have other much stronger biological drivers than their known distinct epigenetic drivers, or our sample size is too small to detect transcriptional differences.
```

> We next performed gene set variant analysis (GSVA) for Hallmark cancer gene sets to demonstrate activation of underlying oncogenic pathways (**Figure {@fig:Fig5}B**, and we characterized the immune cell landscape across OpenPBTA tumors using the quanTIseq method which estimates absolute fractions of immune cell types (**Figure {@fig:Fig5}C** and **Figure {@fig:S6}E**).

Reviewer: This sentence might be too long now. How about:
```suggestion
We next performed gene set variant analysis (GSVA) for Hallmark cancer gene sets to demonstrate activation of underlying oncogenic pathways (**Figure {@fig:Fig5}B** and quantified immune cell fractions across OpenPBTA tumors using quanTIseq (**Figure {@fig:Fig5}C** and **Figure {@fig:S6}E**).
```

> These analyses recapitulated several previously-described trends.

Reviewer: ```suggestion
Through these analyses, we were able to recapitulate previously-described tumor biology.
```

For example, HGG, DMG, MB, and ATRT tumors are known to upregulate _MYC_ [@doi:10.3390/genes8040107] which in turn activates _E2F_ and S phase [@pmid:11511364].

> Indeed, we detected significant (Bonferroni-corrected p < 0.05) upregulation of _MYC_ and _E2F_ targets, as well as G2M (cell cycle phase following S phase) in both embryonal and diffuse astrocytic and oligodendroglial tumors compared to several other cancer groups.

Reviewer: Is "Bonferroni-corrected" is redundant, ie, "corrected" is understood?

Author: _No, there are actually _many_ different ways to correct p-values! Just saying "corrected" along does not imply Bonferroni._

Reviewer: oh, no I meant, saying "Bonferroni" == it has been corrected :)

In contrast, low-grade astrocytic tumors showed significant downregulation (Bonferroni-corrected p < 0.05) of these pathways.
Schwannomas and neurofibromas, which have a documented inflammatory immune microenvironment of T and B lymphocytes as well as tumor-associated macrophages (TAMs), are driven by upregulation of cytokines such as IFN$\gamma$, IL-1, and IL-6, and TNF$\alpha$ [@doi:10.1093/noajnl/vdaa023].

> We observed this trend in GSVA hallmark pathways (Bonferroni-corrected p < 0.05) (**Figure {@fig:Fig5}B**) and immune cell deconvolution, which showed that monocytes dominated the identified immune cells in these tumors (**Figure {@fig:Fig5}C**).

Reviewer: ```suggestion
Indeed, we observed significant upregulation of these cytokines in GSVA hallmark pathways (Bonferroni-corrected p < 0.05) (**Figure {@fig:Fig5}B**) and found immune cell types dominated by monocytes in these tumors (**Figure {@fig:Fig5}C**).
```

I think of "trend" as synonymous with "not-significant/near-significant", so maybe we can reword

We also observed significant upregulation of pro-inflammatory cytokines IFN$\alpha$ and IFN$\gamma$ in LGG and craniopharyngiomas compared to medulloblastoma and ependymoma tumors (Bonferroni-corrected p < 0.05), both of which showed significant down-regulation of these cytokines (**Figure {@fig:Fig5}B**).
Together, these results supported previous proteogenomic findings of lower immune infiltration in aggressive medulloblastomas and ependymomas versus higher immune infiltration in _BRAF_-driven LGG and craniopharyngiomas [@doi:10.1016/j.cell.2020.10.044]. 

Although CD8+ T-cell infiltration across all cancer groups was quite low (**Figure {@fig:Fig5}C**), we observed some signal in specific cancer molecular subtypes (medulloblastoma Groups 3 and 4) as well as outlier tumors (BRAF-driven LGG, BRAF-driven and wildtype ganglioglioma, H3 wildtype HGG; **Figure {@fig:S6}E**) 

> Surprisingly, the classically immunologically-cold HGG and DMG tumors [@doi:10.1186/s40478-018-0553-x; 10.1093/brain/awab155] contained higher overall fractions of immune cells, with monocytes, dendritic cells, and NK cells the most prevalent (**Figure {@fig:Fig5}C**).

Reviewer: ```suggestion
Surprisingly, the classically immunologically-cold HGG and DMG tumors [@doi:10.1186/s40478-018-0553-x; 10.1093/brain/awab155] contained higher overall fractions of immune cells, with monocytes, dendritic cells, and NK cells being the most prevalent (**Figure {@fig:Fig5}C**).
```

"being" or "as"?

Reviewer: 👍

> This finding suggests that quanTIseq might actually be capturing microglia within these immune cell fractions.

Reviewer: ```suggestion
Thus, we suspect that quanTIseq might actually be capturing microglia within these immune cell fractions.
```

try to mix it up with not saying "this finding" or similar so many times and this brings it back to a present tense, what we think/hypothesize vs a plain statement of results

> While we did not detect notable prognostic effects of immune cell infiltration in HGG or DMG cancer groups, we did find that high levels of macrophage M1 and monocytes were associated with poorer overall survival (monocyte HR = 5.2e17, 95% CI = 1.2e5 - 2.3e30, p = 0.006) in medulloblastoma tumors (**Figure {@fig:Fig5}D**).

Reviewer: ```suggestion
While we did not detect notable prognostic effects of immune cell infiltration in HGG or DMG tumors, we did find that high levels of macrophage M1 and monocytes were associated with poorer overall survival (monocyte HR = 5.2e17, 95% CI = 1.2e5 - 2.3e30, p = 0.006) in medulloblastoma tumors (**Figure {@fig:Fig5}D**).
```

We further reproduced previous findings (**Figure {@fig:Fig5}E**) that medulloblastomas typically have low expression of _CD274_ (PD-L1) [@doi:10.18632/oncotarget.24951].

> Consistent with a previous study [@doi:10.1080/2162402X.2018.1462430], we found higher expression of _CD274_ in the WNT subtype, which has the best prognosis of all medulloblastoma subgroups, compared to all other molecular subtypes (**Figure {@fig:Fig5}E**).
> However, we also found that higher expression of _CD274_ was significantly, although marginally, associated with improved overall prognosis (HR = 0.00074, 95% CI = 3.0e-6 - 0.18, p = 0.01) for medulloblastoma samples (**Figure {@fig:Fig5}D**)

Reviewer: I had this written the other way because we found the survival result, and it can be explained by the previous literature on WNT subtypes, so we should probably write it just as that "can be explained by". 
```suggestion
However, we also found that higher expression of _CD274_ was significantly associated with improved overall prognosis (HR = 0.00074, 95% CI = 3.0e-6 - 0.18, p = 0.01) for medulloblastoma samples (**Figure {@fig:Fig5}D**)
This may be explained by the higher expression of _CD274_ found in WNT subtype tumors by us and others [@doi:10.1080/2162402X.2018.1462430], as this diagnosis carries the best prognosis of all medulloblastoma subgroups (**Figure {@fig:Fig5}E**).
```

I don't really understand the word "marginally" here - significant is significant based on the alpha level you set out to test against. In the past, I had also read that one should never use adjectives to describe significance (eg - my grad PI used to use the words "highly significant" for QTL which passed a specific stat threshold, which I refused to ever put in my dissertation, haha!) Ref [here](https://www.statisticsdonewrong.com/) 
>The p value is a measure of surprise, not a measure of the size of the effect.

Reviewer: oh, the HR! missed that. got it.

> Finally, we asked whether any molecular subtypes might show an immunologically-hot phenotype, as roughly defined by a greater proportion of CD8+ to CD4+ T cells. **TODO: reference for this ratio?**

Reviewer: Ha, ref is Jess Foster.... I can ask!

Reviewer: Missed this again also - We can use this paper: [10.1136/jitc-2021-004012](http://dx.doi.org/10.1136/jitc-2021-004012)

Reviewer: Also, this one: [10.4236/jct.2013.48164](http://dx.doi.org/10.4236/jct.2013.48164)
